src/hooks/events.ts

```diff
@@ -0,0 +1,42 @@
+import { useMemo } from "react";
+
+import { useSelector } from "./useSelector";
+import { useVenueId } from "./useVenueId";
+import { useFirestoreConnect, isLoaded } from "./useFirestoreConnect";
+import { useSovereignVenueId } from "./useSovereignVenueId";
+
+export const useConnectWorldEvents = () => {
+  const venueId = useVenueId();
+
+  const { sovereignVenueId, isSovereignVenueIdLoading } = useSovereignVenueId(
+    venueId
+  );
+
+  useFirestoreConnect(() => {
+    if (isSovereignVenueIdLoading || !sovereignVenueId) return [];
+
+    return [
+      {
+        collectionGroup: "events",
+        where: [["worldId", "==", sovereignVenueId]],
+        storeAs: "worldEvents",
+      },
+    ];
+  });
+};
+
+export const useWorldEvents = () => {
+  useConnectWorldEvents();
+
+  const selectedWorldEvents = useSelector(
+    (state) => state.firestore.ordered.worldEvents
+  );
```

Comment:
If we stick with using redux state like this, then this selector should be extracted into `src/utils/selectors.ts`.